Earthfile

```
@@ -0,0 +1,444 @@
+# See https://docs.earthly.dev/docs/earthfile/features
+VERSION --try --raw-output 0.8
+
+PROJECT crossplane/crossplane
```

not sure what this does, but should probably be:
```suggestion
PROJECT upbound/crossplane
```